pyshacl/rdfutil/load.py

```diff
@@ -139,7 +139,7 @@ def load_from_source(
     :return:
     """
     source_is_graph = False
-    open_source: Optional[BufferedIOBase] = None
+    open_source: Optional[Union[BufferedIOBase, BinaryIO]] = None
```

Comment:
What is the purpose of `BinaryIO` in here?
I added the `cast` in the previous release specifically to avoid the use of `BinaryIO` in `load.py`.

Reply:
I added `BinaryIO` per a suggestion from `make type-check`.  There was still a type influence from an `open(..., 'rb')` call.  With your note on using `cast` to avoid `BinaryIO`, I've pushed another patch.